src/app.js

```diff
@@ -4,12 +4,29 @@
  */
 
 import { SharedMap } from "fluid-framework";
+import { AzureClient } from "@fluidframework/azure-client";
+import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
 import { TinyliciousClient } from "@fluidframework/tinylicious-client";
 
+const useAzure = process.env.FLUID_CLIENT === "azure";
+
 export const diceValueKey = "dice-value-key";
 // Load container and render the app
 
-const client = new TinyliciousClient();
+const clientProps = {
+    connection: {
+        type: "remote",
+        tenantId: "" /*REPLACE WITH YOUR TENANT ID*/,
+        tokenProvider: new InsecureTokenProvider("" /*REPLACE WITH YOUR PRIMARY KEY*/, {
+            userId: "userId",
+            userName: "Test User",
+        }),
+        endpoint: "" /*REPLACE WITH YOUR AZURE ENDPOINT*/,
+    },
+};
+
+const client = useAzure ? new AzureClient(clientProps) : new TinyliciousClient();
```

Comment:
probably good to mimic it as we do in the external-controller example README: https://github.com/microsoft/FluidFramework/blob/main/azure/packages/external-controller/README.md

```ts
const connectionConfig: AzureConnectionConfig = useAzure
	? {
			type: "remote",
			tenantId: "YOUR-TENANT-ID-HERE",
			tokenProvider: new InsecureTokenProvider("YOUR-SECRET-HERE", user),
			endpoint: "ENTER-DISCOVERY-ENDPOINT-URL-HERE",
	  }
	: {
			type: "local",
			tokenProvider: new InsecureTokenProvider("fooBar", user),
			endpoint: "http://localhost:7070", 
```suggestion
const client = useAzure ? new AzureClient(clientProps) : new TinyliciousClient();
```
```suggestion
const client = useAzure ? new AzureClient(clientProps) : new TinyliciousClient();
```
	  };
```